codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/PythonServerCodegenVisitor.kt

```diff
@@ -164,4 +167,11 @@ class PythonServerCodegenVisitor(
         )
             .render()
     }
+
+    override fun operationShape(shape: OperationShape) {
```

Comment:
This used to live in `operation_handler.rs` but since we removed this module we opt to generate it via the `CodegenVisitor`.